### ISSUE TITLE
Short term backlog is too full -> how to keep number of issues lower eg. 100 issues

### ISSUE THREAD

**Issue by mattab** [piwik/piwik#6430]
The goal of this issue is to keep the number of issues in `Short term` milestone ideally under 100 tickets. 

As part of our [effort to focus on Reliability, Security and Performance](http://piwik.org/blog/2014/10/announcement-piwik-focus-reliability-performance-security/) we want to keep our Short Term milestone under control with a limited and realistic number of issues. This will help maintain our small team agile going forward. There are several problems with long backlog such as inefficiency going through the backlog, triaging, and general lack of focus. This must be changed before it becomes too big an issue.

**Short term milestone** as of 2014 Oct 13th:

 * 222 issues
 * 48 issues `Major`
 * 114 issues `Bug`
 * 46 issues `Task`
 * 64 issues `Enhancement`

**Proposed steps**
 * for issues `Bug`:
  * Triage all Bugs ([click to see the list](https://github.com/piwik/piwik/issues?q=is%3Aopen+is%3Aissue+label%3ABug+milestone%3A%22Short+term%22+)) and assess whether they are still valid, reproducible, whether they are really important (all our bugs are in `Short term`)
  * We shall schedule the remaining Bugs to be fixed in next 3-5 releases. Ideally we will come back to our beloved "No bug policy" in Piwik.
 * for issues `Task` and `Enhancement`
  * are there issues not tagged `Major` ([click to see the list](https://github.com/piwik/piwik/issues?q=is%3Aopen+is%3Aissue+-label%3ABug+-label%3AMajor+milestone%3A%22Short+term%22)) that could be moved to Mid or Long term?

**Comment by mattab** [comment]
quick update: somehow we can't get down to 100 issues in Short term - 3 months after my last comment here we still have 216 open issues still in Short term.

**Comment by mattab** [comment]
we need to do something about this: having a realistic `Short term` backlog is very important for the product engineering team. 

As a first step, I will go through the backlog and move as many issues as possible out of `Short term` - for example, for the first time I will move `bugs` to `Mid term`. This will be the end of our "no bug policy" but it wasn't fair anyway to announce a No-bug policy and then not actually do it.

In Short term will be the most important bugs as well as 1-3 features from our [Roadmap](http://piwik.org/roadmap/) (see also #7624)

**Comment by mattab** [comment]
there were 216 issues, I did a first pass over moving the least important issues out to Mid term, and we now got 125 issues in Short term

Plus there 110 ungroomed issues with no milestone and likely a few more for Short term.

I'll take another look later how we can make it under 100

**Comment by Globulopolis** [comment]
Maybe split "Bug" into two section? Critical or not. First should be fixed as soon as possible, second can be moved to "short term bug list".

**Comment by mattab** [comment]
yes we use `Major` and `Critical` labels for marking important and occasional critical issues.

**Comment by mattab** [comment]
we made it: 100 issues in the `Short term` backlog!!

![100 issues](https://cloud.githubusercontent.com/assets/466765/7060609/6093a348-ded8-11e4-8521-8594fd557d28.png)

Out of those:
 * type breakdown: 42 bugs, 28 enhancements, 26 tasks, 4 pull requests
 * [18 Major issues](https://github.com/piwik/piwik/issues?q=is%3Aopen+milestone%3A%22Short+term%22+label%3AMajor)

Job done - this will help a lot users, team members, and product manager, to know/plan for what to work on next  :+1: 

next step: create the public Roadmap page: #7624 and also related: release a document explaining our vision and values: #7376
